src/Microsoft.VisualStudio.Editors/PublicAPI.Shipped.txt

```diff
@@ -8,9 +8,10 @@ Const Microsoft.VisualStudio.Editors.PropertyPages.ChildPageSite.NestingCharacte
 Const Microsoft.VisualStudio.Editors.SettingsDesigner.PublicSettingsSingleFileGenerator.SingleFileGeneratorName = "PublicSettingsSingleFileGenerator" -> String
 Const Microsoft.VisualStudio.Editors.SettingsDesigner.SettingsSingleFileGenerator.SingleFileGeneratorName = "SettingsSingleFileGenerator" -> String
 LogLevel
-LogLevel.Info = 1 -> LogLevel
+LogLevel.Info = 2 -> LogLevel
+LogLevel.Minimal = 1 -> LogLevel
 LogLevel.None = 0 -> LogLevel
-LogLevel.Verbose = 2 -> LogLevel
+LogLevel.Verbose = 3 -> LogLevel
```

Comment:
This is a breaking change in our public API.

Does this type need to be public? Do we know who else is using it?

Reply:
Much of the editors is public, and likely could be made private/internal. In this case, a customer would have had to taken a dependency upon an enum that's only used in the presentation of a dropdown list on a settings page. I very much doubt this would hit anyone, but I don't know how strict we need to be.

Perhaps we just make the enum internal. What do you think?

Reply:
To make this enum internal (friend) we must also make `Microsoft.VisualStudio.Editors.OptionPages.GeneralOptions` internal. I think this is reasonable. It has two properties, both relating to the up-to-date check.